extensions/azurecore/src/account-provider/azureAccountProviderService.ts

```diff
@@ -142,18 +143,28 @@ export class AzureAccountProviderService implements vscode.Disposable {
 			// Case 2: Provider was enabled and is now disabled - unregister provider
 			if (oldConfigValue && !newConfigValue) {
 				providerChanges.push(this.unregisterAccountProvider(provider));
+				this.activeProviderCount--;
 			}
 
 			// Case 3: Provider was disabled and is now enabled - register provider
 			if (!oldConfigValue && newConfigValue) {
 				providerChanges.push(this.registerAccountProvider(provider));
+				this.activeProviderCount++;
 			}
 
 			// Case 4: Provider was added from JSON - register provider
-			if (provider.configKey !== 'enablePublicCloud' && provider.configKey !== 'enableUsGovCloud' && provider.configKey !== 'enableChinaCloud') {
+			if (provider.configKey !== loc.enablePublicCloudCamel && provider.configKey !== loc.enableUsGovCloudCamel && provider.configKey !== loc.enableChinaCloudCamel) {
```

Comment:
These need not be localized - since they're config keys - so you can use constants instead.